### DISCUSSION TITLE
Ruin scan missions do not show progress bar in dedicated server

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#15421] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

Ruin scan missions do not show progress bar in dedicated server

### Reproduction steps

.

### Bug prevalence

Just once

### Single player or multiplayer?

Multiplayer hosted using a dedicated server

### Version

v1.7.7.0 (Winter Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! I'm afraid I could not reproduce the issue, the progress bar seems to appear in my tests:

https://github.com/user-attachments/assets/a1a46148-b0f9-4809-b5f1-a7106c138ad5

I wonder if some of the mods you have installed could have something to do with the issue?

**Comment by A2ur3** [comment]
it seems the issue is caused by hiding boss healthbar

**Comment by Regalis11** [comment]
This has been fixed in our private repo now, fix coming up in the next unstable build and the upcoming release.
